recipes/simple-azure-blob-downloader/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "simple-azure-blob-downloader" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/simple-azure-blob-downloader-{{ version }}.tar.gz
+  sha256: e8b07282a4bbd21e56220b5cd8e973ff8a20d0d47f10c348489cc010713d1d23
+
+build:
+  entry_points:
+    - download-azure-blob = simple_azure_blob_downloader.download_azure_blob:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >={{ python_min }}
+    - azure-identity >=1.10.0
+    - azure-storage-blob >=12.12.0
+
+test:
+  imports:
+    - simple_azure_blob_downloader
+  commands:
+    - pip check
+    - download-azure-blob --help
+  requires:
+    - pip
+    - python {{ python_min }}
+
+about:
+  home: 'https://outerbounds.co/'
```

Comment:
I could not find anything related to this project in that webpage. Why not use https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob that is listed in the PyPI metadata?

Reply:
the provided url is the authors website for a lack of a better one. The package is a wrapper on the azure blob storage in order to provide a simplified CLI for specific use-cases.

I can try to come up with a better url next week if the current one is not acceptable. As the linter complains about the lack of one, presumably providing a `home:` is required?